src/v/pandaproxy/rest/proxy.h

```diff
@@ -31,21 +32,24 @@ class proxy {
       const YAML::Node& config,
       ss::smp_service_group smp_sg,
       size_t max_memory,
-      ss::sharded<kafka::client::client>& client);
+      ss::sharded<kafka::client::client>& client,
+      cluster::controller* controller);
```

Comment:
```
  ss::sharded<kafka::client::client>& client)
  ss::sharded<kafka::client::client>& client,
  sharded_client_cache& client_cache,
  cluster::controller* controller)
```

i think this commit may not compile. up above in proxy.cc it looks like the constructor takes a client_Cache reference but here in the header the constructor doesn't have that argument.